### PR TITLE
Site overhaul

### PR DESCRIPTION
**Pull request by 404invalid-user**

### CODE REVIEW

**Comment by cobaltgit** [comment]
Damn I can't merge it
image

File: good-styles.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+html,
+body {
+    min-height: 100%;
+    margin: 0;
+    color: honeydew;
+    background-color: rgb(128, 126, 126);
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (sabianroberts) [inline good-styles.css]: You need to add a webfont kit for Helvetica since only Macs come with the font already installed.

File: index.html [hunk 2]
```diff
@@ -1,33 +1,50 @@
...
 <body>
-  <h1>OldDigibox Website Portal</h1>
-  <a href="https://olddigibox.com">OldDigibox</a> <p>Main website</p>
-  <a href="https://olddigibox.github.io/sky-web-epg">Sky Web EPG</a> <p>A web browser re-creation of the Sky EPG TV Guide.</p>
-  <a href="https://brysonrecords.bandcamp.com/album/sky-web-epg">Sky Web EPG - The Album</a> <p>This album contains the tracks added exclusively to the Sky Web EPG! <br />
-    Buy now for <strong>£7.99!</strong></p>
-  <a href="https://brysonrecords.bandcamp.com/album/sky-web-epg-2">Sky Web EPG 2 (INCOMPLETE)</a> <p>An extra collection of exclusive tracks for the Sky Web EPG. <br /> Get it free before it releases on stores! </p>
+    <center>
+        <div class="logo-contain">
+            <img class="logo-img" src="olddigibox.png" width="160" height="60" />
```
Comment (sabianroberts) [inline index.html:15]: The file "olddigibox.png" isn't in your fork; did you forget to add it?

**Review by sabianroberts — CHANGES_REQUESTED** [review]
Helvetica webfont kit needs adding and olddigibox.png (see comments on files)

**Comment by sabianroberts** [comment]
> Damn I can't merge it
> image

I've requested some changes; you need to read the code carefully rather than looking at it once and going "oh that looks nice, I'll merge"

There's just a missing webfont kit and an image I think

File: good-styles.css (hunk 1 above)
Comment (404invalid-user) [reply good-styles.css]: I use a mac I didn't know that. I have removed that font and added another

File: index.html (hunk 2 above)
Comment (404invalid-user) [reply index.html:15]: Yes it is there now

**Comment by 404invalid-user** [comment]
Should be done now

**Review by sabianroberts — APPROVED** [review]

**Pull request merged**